scripts/raspberry_pi_setup.sh

```diff
@@ -1,18 +1,32 @@
 #!/bin/bash
 
-SCRIPT_DIR=`dirname "${BASH_SOURCE[0]}"`
-if [ -e "$SCRIPT_DIR/python.sh" ]; then
-    KALITE_DIR=$SCRIPT_DIR/../kalite
-else
-    KALITE_DIR=$SCRIPT_DIR/kalite
```

Comment:
This block was no longer necessary because the script is not supposed to be run in a git checkout.